Resources/Private/JavaScript/components/RedirectForm.tsx

```diff
@@ -0,0 +1,320 @@
+                selected={dateTime}
+                timeCaption={translate('datepicker.time', 'Time')}
+                onChange={(value) => this.handleDatePickerChange(property, value)}/>
+        );
+    };
+
+    /**
+     * Renders list of changed redirects to be used in a flash message
+     *
+     * @param changedRedirects
+     */
+    private renderChangedRedirects = (changedRedirects: Array<Redirect>): string => {
+        const {translate} = this.props;
+        return (
+            '<p>' + translate('message.relatedChanges', 'Related changes') + '</p>' +
+            '<ul>' +
+            changedRedirects.map((redirect, index) => (
+                '<li>' + (redirect.host || '') + '/' + redirect.sourceUriPath + '&rarr;' + redirect.targetUriPath + '</li>'
+            )).join('') +
+            '</ul>'
+        );
+    };
+
+    render() {
+        const {
+            translate,
+            redirect,
+            statusCodes,
+            idPrefix,
+            validSourceUriPathPattern,
+            handleCancelAction,
+        } = this.props;
+
+        const {
+            host,
+            sourceUriPath,
+            targetUriPath,
+            statusCode,
+            startDateTime,
+            endDateTime,
+            comment,
+            isSendingData,
+        } = this.state;
+
+        return (
+            <form onSubmit={e => this.handleSubmit(e)} className="add-redirect-form">
+                <div className="row">
+                    <div className="neos-control-group">
+                        <label className="neos-control-label" htmlFor={idPrefix + 'host'}>{translate('host')}</label>
+                        <input name="host" id={idPrefix + 'host'} type="text"
+                               placeholder="www.example.org" value={host || ''} onChange={this.handleInputChange}/>
+                    </div>
+                    <div className="neos-control-group">
+                        <label className="neos-control-label"
+                               htmlFor={idPrefix + 'sourceUriPath'}>{translate('sourceUriPath')}*</label>
+                        <input name="sourceUriPath" id={idPrefix + 'sourceUriPath'} type="text"
+                               title={validSourceUriPathPattern} onChange={this.handleInputChange}
+                               autoFocus={true} required={true} placeholder="the-old-url/product-a"
+                               pattern={validSourceUriPathPattern} value={sourceUriPath || ''}/>
+                    </div>
+                    <div className="neos-control-group">
+                        <label className="neos-control-label"
+                               htmlFor={idPrefix + 'statusCode'}>{translate('statusCode')}</label>
+                        <select name="statusCode" id={idPrefix + 'statusCode'} value={statusCode}
+                                onChange={this.handleInputChange}>
+                            {Object.keys(statusCodes).map((code) => (
```

Comment:
You prefer
`(code) => (`
or
`code => (`
?

Reply:
My IDE always adds the braces and sometimes I simply forget removing them ;)
So I prefer without. Thanks